src/libs/common/test/string_compare.test.cpp

```diff
@@ -0,0 +1,75 @@
+#include "storm/string_compare.hpp"
+
+#include <catch2/catch.hpp>
+
+using namespace storm;
```

Comment:
if this possible to move these two 'using namespace' inside TEST_CASE?
if so, do that, please

Reply:
sure :+1: